docs/examples/tf/pets_training/train.py

```diff
@@ -21,36 +21,74 @@
 import os, math
 os.environ['TF_CPP_MIN_LOG_LEVEL'] = '3'
```

Comment:
@LakshmiKumar23 Add `os.environ['TF_FORCE_GPU_ALLOW_GROWTH'] = 'true'` to limit TF GPU memory consumption. RPP throws hip mem allocation errors if the env variable is not set in ROCm TF docker
`Runtime error: hipMalloc(&(this->initHandle->mem.mgpu.scratchBufferHip.floatmem), sizeof(Rpp32f) * 372877312) returned 2 at /data/unittests_qa/rpp/src/modules/hip/handlehip.cpp:256`